sycl/test/esimd/simd_view.cpp

```diff
@@ -69,6 +69,12 @@ SYCL_ESIMD_FUNCTION void test_simd_view_copy_ctor() {
   auto v0_view_copy(v0_view);
 }
 
+// test construction from vector.
+SYCL_ESIMD_FUNCTION void test_simd_view_from_vector() {
+  simd<int, 16> s = 0;
+  simd_view v = s;
```

Comment:
please add a test case like simd_view v(s);
Can this be disabled and diagnosed - simd_view v(simd<int,8>()); ?

Reply:
Added both test cases.